### ISSUE TITLE
Site Request: Central Texas Food Bank

### ISSUE THREAD

**Issue by jesusnunezxoco** [jadkins89/Recipe-Scraper#21]
Hello! I'm making a website for a local food bank, and I would like their recipes to be supported by this API. Below is the link:

https://www.centraltexasfoodbank.org/recipes

PS: I REALLY appreciate this API!!!

**Comment by jadkins89** [comment]
It's been added! Let me know if you run into any issues using it. I just published the update to npm.

**Comment by jesusnunezxoco** [comment]
Thank you! It's worked for several of the recipes, but I'm getting an error for some of them.  Below are the particular recipes I'm having trouble with:

https://www.centraltexasfoodbank.org/recipe/garden-avocado-dip
https://www.centraltexasfoodbank.org/recipe/lentil-tacos
https://www.centraltexasfoodbank.org/recipe/grilled-cheese-mushroom-spinach-and-tomato
https://www.centraltexasfoodbank.org/recipe/turnips-olives

Here is the error I get:

> Error: No recipe found on page
    at Request._callback (/home/jesusnunezxoco/coding/jnx/playground_nodeJS/server/node_modules/recipe-scraper/scrapers/centraltexasfoodbank.js:53:20)
    at Request.self.callback (/home/jesusnunezxoco/coding/jnx/playground_nodeJS/server/node_modules/request/request.js:185:22)
    at Request.emit (events.js:315:20)
    at Request.<anonymous> (/home/jesusnunezxoco/coding/jnx/playground_nodeJS/server/node_modules/request/request.js:1154:10)
    at Request.emit (events.js:315:20)
    at IncomingMessage.<anonymous> (/home/jesusnunezxoco/coding/jnx/playground_nodeJS/server/node_modules/request/request.js:1076:12)
    at Object.onceWrapper (events.js:421:28)
    at IncomingMessage.emit (events.js:327:22)
    at endReadableNT (_stream_readable.js:1221:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
[nodemon] restarting due to changes...
[nodemon] starting `node index.js`
Listen on port 3000
Error: No recipe found on page
    at Request._callback (/home/jesusnunezxoco/coding/jnx/playground_nodeJS/server/node_modules/recipe-scraper/scrapers/centraltexasfoodbank.js:53:20)
    at Request.self.callback (/home/jesusnunezxoco/coding/jnx/playground_nodeJS/server/node_modules/request/request.js:185:22)
    at Request.emit (events.js:315:20)
    at Request.<anonymous> (/home/jesusnunezxoco/coding/jnx/playground_nodeJS/server/node_modules/request/request.js:1154:10)
    at Request.emit (events.js:315:20)
    at IncomingMessage.<anonymous> (/home/jesusnunezxoco/coding/jnx/playground_nodeJS/server/node_modules/request/request.js:1076:12)
    at Object.onceWrapper (events.js:421:28)
    at IncomingMessage.emit (events.js:327:22)
    at endReadableNT (_stream_readable.js:1221:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)

**Comment by jadkins89** [comment]
Thanks for troubleshooting the scraper. I was worried about that being the case since the site doesn't seem to follow the same pattern everywhere. It looks like there are 3 different arrangements of mark up for the instructions portion of the recipes (at least from the samples we've looked at). I updated the logic to handle all of these scenarios. Hopefully that covers everything! Let me know if you find something else not working as intended.
